Formula/g/gnupg.rb

```diff
@@ -1,13 +1,13 @@
 class Gnupg < Formula
   desc "GNU Pretty Good Privacy (PGP) package"
   homepage "https://gnupg.org/"
-  url "https://gnupg.org/ftp/gcrypt/gnupg/gnupg-2.4.6.tar.bz2"
-  sha256 "95acfafda7004924a6f5c901677f15ac1bda2754511d973bb4523e8dd840e17a"
+  url "https://gnupg.org/ftp/gcrypt/gnupg/gnupg-2.4.7.tar.bz2"
+  sha256 "7b24706e4da7e0e3b06ca068231027401f238102c41c909631349dcc3b85eb46"
   license "GPL-3.0-or-later"
 
   livecheck do
     url "https://gnupg.org/ftp/gcrypt/gnupg/"
-    regex(/href=.*?gnupg[._-]v?(\d+(?:\.\d+)+)\.t/i)
+    regex(/href=.*?gnupg[._-]v?(2\.[46](?:\.\d+)+)\.t/i)
```

Comment:
why changing this livecheck?

Reply:
GnuPG numbers "devel" branches with even minor numbers, i.e. 2.0, 2.2, 2.4, 2.6, ...
At the moment, livecheck returns 2.5.1 which is _really_ explosively unstable.
We don't want to pick this version not only because of project policy, but also due to common sense.
See: https://gnupg.org/download/index.html#end-of-life

Reply:
got you, thanks!